bitrixSM/internal/transport/handlers/createTaskr.go

```diff
@@ -56,3 +66,68 @@ func CreateTask(c *gin.Context) error {
 	c.JSON(resp.StatusCode, gin.H{"message": "task created successfully"})
 	return nil
 }
+
+func parseTaskParams(userID int64, msg string) (*createTaskParams, error) {
+	params := make(map[string]string)
+	parts := strings.Fields(msg)
+
+	if len(parts) < 1 {
+		return nil, fmt.Errorf("empty command")
+	}
+
+	for _, part := range parts[1:] {
```

Comment:
I see you ignoring first word, but for business logic is unclear, and code is not helping with this transparency. You can use additional semantic functions to encapsulate specific logic/

Reply:
if it is not clear - let's talk about it. Just ask me in the way to have a new meet